common/timedcache/timedcache.go

```diff
@@ -25,18 +25,55 @@ func (te *timedEntry) expired() bool {
 // to expire at exactly the ttl time. The expiration mechanism is 'lazy', and
 // will only remove expired objects at next access.
 type TimedCache struct {
-	ttl   int64     // Time (in seconds) each entry is allowed to live for
-	cache lru.Cache // Underlying size-limited LRU cache
```

Comment:
Are these comments still relevant?

Reply:
I felt like they were noisy without communicating useful information, but I can add them back.

Reply:
I see, it is a bit verbose as some of the same info is encapsulated in the comment above. The only new information is that `ttl` is measured in seconds, which is good to know. This could be added to the paragraph above or left as a standalone comment.